es6 & beyond/ch4.md

```diff
@@ -69,21 +70,21 @@ function ajax(url) {
 ajax( "http://some.url.1" )
 .then(
 	function fulfilled(contents){
-		// handle `contents` success
+		// manipula o `conteúdo` no sucesso
 	},
 	function rejected(reason){
-		// handle ajax error reason
+		// manipula a razão do erro da requisição
 	}
 );
 ```
 
-Promises have a `then(..)` method that accepts one or two callback functions. The first function (if present) is treated as the handler to call if the promise is fulfilled successfully. The second function (if present) is treated as the handler to call if the promise is rejected explicitly, or if any error/exception is caught during resolution.
+Promises possuiem um metodo chamado `then(..)` que aceita uma ou duas funções como callback. A primeira função (se presente) é tratata como o manipulador de uma promise cumprida com sucesso. A segunda função (se presente) é tratada como o manipulador se a primise for explicitamente rejeitada, ou se algum erro ou exeção ocorrer durante a resolução.
 
-If one of the arguments is omitted or otherwise not a valid function -- typically you'll use `null` instead -- a default placeholder equivalent is used. The default success callback passes its fulfillment value along and the default error callback propagates its rejection reason along.
+Se um dos argumentos for omitido ou se não for uma função válida -- normalmente você usará `null` -- um valor padrão equivalente é usado. O callback padrão de sucesso passa seu valor de cumprimento adiante e a função callback padrão de erro propaga seu valor de rejeição adiante.
```

Comment:
Se um dos argumentos for omitido ou se não for uma função válida -- normalmente você usará `null` **em vez isso**